pandas/core/indexes/datetimes.py

```diff
@@ -1991,15 +1992,17 @@ def tz_localize(self, tz, ambiguous='raise', errors='raise'):
         DatetimeIndex(['2018-03-01', '2018-03-02', '2018-03-03'],
```

Comment:
`Returns` should contain a description. See the example in: https://python-sprints.github.io/pandas/guide/pandas_docstring.html#section-4-returns-or-yields